src/Configurator/EnvConfigurator.php

```diff
@@ -142,4 +138,21 @@ private function unconfigurePhpUnit(Recipe $recipe, $vars)
             file_put_contents($phpunit, $contents);
         }
     }
+
+    private function replaceGeneratedValue($value)
+    {
+        if ('%generate(secret)%' === $value) {
+            return $this->generateRandomBytes();
+        }
+        if (preg_match('~^%generate\(([0-9]+)\)%$~', $value, $matches)) {
```

Comment:
I would say we need `generate(secret, %d+)` here instead (as generate() might generate some else than secret at some point).